src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,49 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health) {
+    super(name, health);
+
+    this.role = 'herbivore';
+    this.hidden = false;
```

Comment:
```suggestion

    this.hidden = !hidden;
```

Reply:
<img width="572" alt="image" src="https://user-images.githubusercontent.com/102521367/187884471-84584c83-90a6-431f-8b21-823897856d37.png">

Reply:
this.hidden = !this.hidden;